modules/microshift-manifests-deletion-overview.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * microshift//running_applications/microshift-deleting-resource-manifests.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-manifests-deletion-overview_{context}"]
+= How manifest deletion works
+
+By default, {microshift-short} expects to find deletion manifests in the `delete` subdirectories within the manifests path. When a manifest is placed in these subdirectories, {microshift-short} automatically handles the removal at every start. Read through the following procedure to understand how manifests deletion works in {microshift-short}.
+
+.Procedure
+
+. At every start, before applying the manifests, {microshift-short} scans the following `delete` sub directories within the configured manifests directory to identify the Kustomize manifests that need to be deleted:
+
+* /usr/lib/microshift/manifests/delete
+* /usr/lib/microshift/manifests.d/delete/*
+* /etc/microshift/manifests/delete
+* /etc/microshift/manifests.d/delete/*
+
+. {microshift-short} deletes the resources defined in the kustomize manifests found in the `delete` directories by running the equivalent of `kubectl delete -k` command
```

Comment:
Do we want to include `ignore-not-found` or mention that repeated attempt to delete a manifest does not result in an error?
```suggestion
. {microshift-short} deletes the resources defined in the kustomize manifests found in the `delete` directories by running the equivalent of `kubectl delete --ignore-not-found -k` command
```

Reply:
done. note is added too.